src/main/java/uk/gov/hmcts/reform/translate/model/TranslationsRequest.java

```diff
@@ -0,0 +1,18 @@
+package uk.gov.hmcts.reform.translate.model;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import lombok.NoArgsConstructor;
+
+import java.util.List;
+import javax.validation.constraints.NotEmpty;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+@EqualsAndHashCode
+public class TranslationsRequest {
+    @NotEmpty(message = "Bad Request (001 bad schema)")
```

Comment:
Can we use constants for error messages so we can re-use in swagger and tests.

Reply:
Fixed in 1fba53e73e335d20d33745292ac0bde6f21b513f